recipes/gfflu/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "gfflu" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace("_", "-") }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
` | replace("_", "-")` doesn't do anything

Reply:
Good catch! Removed in 5c7f1d8067